frontend/src/app/admin/change-password/change-password.component.scss

```
@@ -0,0 +1,60 @@
+  color: #0f766e;
+  font-size: 0.85rem;
+}
+
+.actions {
+  margin-top: 10px;
+  display: flex;
+  gap: 10px;
+  justify-content: flex-end;
+}
+
+@media (max-width: 640px) {
```

Stylelint (17.2.0)

[error] Expected "context" media feature range notation (media-feature-range-notation)

(media-feature-range-notation)